scripts/init_chain.sh

```diff
@@ -2,6 +2,46 @@
 # make install-all
 killall -9 lavad
 
+# Function to check if a command is available
+command_exists() {
+    command -v "$1" >/dev/null 2>&1
+}
+
+# Flag to track if jq installation is successful
+jq_installed=false
+
+# Check if jq is not installed
+if ! command_exists jq; then
```

Comment:
IMO this script should _not_ modify the underlying system. first and foremost, it breaks a user's expectation. second, users usually won't run it as root. and third, it anyhow does not (and will not) cover all possible OS flavors.

if `jq` or any other utility is missing, this script should simply fail with a clear indication of the problem, and how the user could fix it.

instead of auto-install here, add a new script e.g. `init_install.sh` (or whatever) that will check what's missing (for all scripts) and install it, or - if the OS not supported by this script - will indicate what the user needs to do. ideally it will describe what it is about to do and prompt the user for permission prior to doing it. future contributors may enhance it to support more systems.

Reply:
done. see https://github.com/lavanet/lava/commit/9cd756adfddb049652b4976c240d950a733b1a3c

Reply:
also this: https://github.com/lavanet/lava/commit/b3bb435b6edeeeb6fa860e9fd70fb3deef53b3f4
sorry for the two commits